solidity/contracts/isms/MultisigIsm.sol

```diff
@@ -287,10 +288,10 @@ contract MultisigIsm is IMultisigIsm, Ownable {
      * @param _metadata ABI encoded module metadata (see MultisigIsmMetadata.sol)
      * @param _message Formatted Hyperlane message (see Message.sol).
      */
-    function _verifyMerkleProof(
+    function verifyMerkleProof(
```

Comment:
also wondering what the motivation was for making these public

Reply:
was part of benchmarking
will restore